### PR TITLE
Store entity /Still click dummy / Many styles

### PR DESCRIPTION
**Pull request by macebm**
###### Backend:
- Database
- First Entity store
- StoreController
- Store Entity
- Store Repository

###### Frontend:
- Header and Footer component
- GlobalStyle component
- NavigationButton Component
- Store and Stores component
- Colors and FontSizes component
- getStores.js

I hope you can have a look at some of them. The Buttons have no function yet.

Thanks

Best wishes 

Marcel

### CODE REVIEW

File: backend/src/Controller/StoreController.php [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+class StoreController extends AbstractController
+{
+    /**
+     * @Route("/store", methods={"GET"})
+     */
+    public function index (
+        StoreRepository $repository,
+        SerializerInterface $serializer
+    ): JsonResponse {
```
Comment (tine-krueger) [inline backend/src/Controller/StoreController.php:22]: Keep in mind: the serialize might not work, when you have tables with relations

File: backend/src/Repository/StoreRepository.php [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+        $this->_em->persist($store);
+        $this->_em->flush();
+
+        return $store;
+    }
+
+    // /**
+    //  * @return Store[] Returns an array of Store objects
+    //  */
```
Comment (tine-krueger) [inline backend/src/Repository/StoreRepository.php]: you can delete those commented things.

**Review by tine-krueger — COMMENTED** [review]
Looks good! Keep going!

**Review by web-devil — APPROVED** [review]

File: backend/src/Controller/StoreController.php (hunk 1 above)
Comment (macebm) [reply backend/src/Controller/StoreController.php:22]: Thank you i changed it now

**Pull request merged**